tests/embed/test_splitae.py

```diff
@@ -0,0 +1,62 @@
+from multiview.embed.splitae import SplitAE
+import numpy as np
+import sklearn.cluster
+import sklearn.datasets
+from itertools import permutations
+import torch
+
+# These are blobs that PCA can reduce to nice clusters easily, so splitAE should be able to aswell
+# cluster_std is high s.t. any 2 features alone do not make nice seperable blobs
+def test_splitae_blobs():
+    accuracies = []
+    for i in range(10):
+        view1, labels = sklearn.datasets.make_blobs(n_samples=1000, n_features=20, centers=None, cluster_std=10.0, center_box=(-10.0, 10.0), shuffle=True, random_state=None)
```

Comment:
line lengths way too long, i don't know what the standard for this package is

Reply:
We're not requiring pycodestyle for our tests. Imo enabling line-wrap in the editor instead of manually formatting is the way to go, hence the couple long lines here. 

I'm not sure if our line length requirements (80 our so characters) in e.g. splitae.py really made it easier to read. It definitely made it harder to edit, with all the new line breaks and the lining up of stuff horizontally.